### DISCUSSION TITLE
Making "privacy policy" content work

### DISCUSSION THREAD

**Discussion by douglaz** [thuliteio/doks#686] [Q&A] [answered]
A very simple question:

The file https://github.com/h-enk/doks/tree/master/content/en/privacy-policy exists but when I try to access /privacy-policy/ the result is 404 and no file is created on npm run build

How to make it work?

**Comment by h-enk** [comment, marked as answer]
Set `draft: false` in the frontmatter of https://github.com/h-enk/doks/tree/master/content/en/privacy-policy

**Reply by douglaz** [reply]
Of course :man_facepalming: 
Thank you!
